source/assets/css/components/_index.scss

```diff
@@ -0,0 +1,7 @@
+@include sl-breakpoint--medium {
+  .sl-c-index {
+    font-size: 1.3rem;
```

Comment:
This comes out to `20.8` rendered pixels. Can we make it `1.25rem` so it comes out to an even 20?

Reply:
Done. My intutition is that 20px is too small and 21px is too big, though, so I encourage you to reconsider.